flutter/test/native_sdk_integration_test.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+import 'mocks.dart';
+import 'sentry_flutter_options_test.dart';
+
+void main() {
+  group('$NativeSdkIntegration', () {
+    setUp(() {
+      TestWidgetsFlutterBinding.ensureInitialized();
+    });
+
+    test('test default values', () async {
+      final channel = const MethodChannel('initNativeSdk');
+
+      String? methodName;
+      dynamic arguments;
+      channel.setMockMethodCallHandler((call) async {
+        methodName = call.method;
+        arguments = call.arguments;
+      });
+      var sut = Fixture().getSut(channel);
+
+      await sut.call(HubAdapter(), createOptions());
```

Comment:
should `HubAdapter()` be a simple mock or fake? otherwise calls will be forwarded to the sentry static class that has a state

Reply:
NativeSDKIntegration doesn't do anything with the Hub so it doesn't matter what I pass. But yeah, it should probably be Hub, just so that it reads better.

Reply:
good point, lets just either make a dummy fake or mock, so when we read it, we know it does nothing, I thought it was bounded with the HubAdapter behavior somehow.